qtopconf.yaml

```diff
@@ -64,6 +66,18 @@ state_abbreviations:
     s: suspended_of_user
     S: suspended_by_the_queue
     shite: testing_of_user
+  pbs:
```

Comment:
Would be nice if all those ware going back to individual sources with a mechanism similar to `get_mnemonic():`.